zettel-mode.el

```diff
@@ -23,7 +23,8 @@
 (require 'counsel)
 (require 'json)
 
-(defvar neuron-zettelkasten nil)
+(defvar neuron-zettelkasten (expand-file-name "~/.zettle")
```

Comment:
I think this should be called `~/zettelkasten` or `~/.zettelkasten`. The word "zettel" refers to an individual note in the zettelkasten.

Reply:
Done